packages/tasks/src/tasks/image-to-image/about.md

```diff
@@ -53,12 +52,21 @@ await inference.imageToImage({
 });
 ```
 
-## ControlNet
+## Use Cases for Text-to-Image Tasks
+
+### ControlNet
 
 Controlling the outputs of diffusion models only with a text prompt is a challenging problem. ControlNet is a neural network model that provides image-based control to diffusion models. Control images can be edges or other landmarks extracted from a source image.
 
 Many ControlNet models were trained in our community event, JAX Diffusers sprint. You can see the full list of the ControlNet models available [here](https://huggingface.co/spaces/jax-diffusers-event/leaderboard).
 
+### Style Transfer
+
+One of the most popular use cases of image-to-image is style transfer. with style transfer models:
+
+- a normal photo can be converted to a painting in the style of a famous painter.
```

Comment:
Yeah, I just used the original wording but I think it might be, lets change it maybe to-
> a regular photo can be transformed into a variety of artistic styles or genres, such as a watercolor painting, a comic book illustration and more.